test/living-html/postmessage.js

```diff
@@ -0,0 +1,123 @@
+"use strict";
+
```

Comment:
I think it should be possible to add tests that don't do script injection but instead reach directly into the iframe and call its postMessage/set its event handlers. No need to duplicate the existing ones all in that style, but one or two would be nice.